src/app/shared/components/template/components/button/button.component.html

```diff
@@ -25,16 +25,18 @@
         [attr.data-rowname]="_row.name"
       ></plh-template-component>
     </span>
+    <ion-ripple-effect></ion-ripple-effect>
```

Comment:
The `card` variant already had a ripple effect, so this can be removed as it is redundant